### ISSUE TITLE
[FEATURE] alpha_Q2 for punching shear

### ISSUE THREAD

**Issue by PaolaCostanzaMiglietta** [EPFL-ENAC/resslab-astra_82001#24]
Is it possible, for the _ponts dalles_ and upcoming _dalles de roulement_, to add another AE called Punching/Poinçonnement/ (I don't know the German) / Punzonamento for which the alpha_Q2 is = 0.4 instead 0.35? The other coefficients are evaluated as usual.

@Jondalar-64 please confirm this is something we want

**Comment by guilbep** [comment]
@PaolaCostanzaMiglietta Sure, everything possible, but not sure how it's related to the AE, since it won't change the alpha Q value ? 
Also, it was already defined in this issue https://github.com/EPFL-ENAC/resslab-astra_82001/issues/14 (point number 2)

Still not sure how to implement this. But We could have a toggle button that change alphaQ2 from 0.35 to 0.40 where

toggle button off text: No Punching Shear taken into account
toggle button on text:  In the case of Punching shear

Or whatever text you want. @charlottegiseleweil  any suggestions on this ?


Issue duplicated from #14

**Comment by Jondalar-64** [comment]
The issue of pnching shear is related to the type/subtype of bridge, since it has to verified for:
- ponts dalles, I suggest it appears as an additional line: For punching shear checks, use Alpha_Q2 is = 0.4
- for the upcoming dalles de roulement (which is one part of any bridge type, which needs separate checking), I suggest also that it appears as an additional line: For punching shear checks, use Alpha_Q2 is = 0.4
For the other bridge types it should not appear as the engineer is checking other elements of the bridge (not slab punching).

@PaolaCostanzaMiglietta and @guilbep, what do you think ?

**Comment by Dimitrios-astra** [comment]
I agree with the proposition of [Jondalar-64](https://github.com/Jondalar-64)

**Comment by charlottegiseleweil** [comment]
fixed with Alpha local/global. to be docuemnted in #14
